src/schema/v2/me/myCollection.ts

```diff
@@ -70,6 +63,12 @@ export const MyCollection: GraphQLFieldConfig<any, ResolverContext> = {
       description:
         "Exclude artworks that have been purchased on Artsy and automatically added to the collection.",
     },
+    sortByLastAuctionResultDate: {
+      type: GraphQLBoolean,
+      defaultValue: false,
+      description:
+        "Sort by most recent price insight updates and filter out artworks without insights.",
+    },
```

Comment:
I think we should be probably changing this to use the same pattern for sorting as in other connections for a more future-proof approach. I suggest we update this to be an enum with the possible sorting methods. In this case, it will be probably just an enum of one value `LAST_AUCTION_RESULT_DATE_DESC`.

Reply:
Very good idea! I kept it as a flag because it is actually doing two things (sorting and filtering (if we don't filter, we get issues in the frontend with pagination)) and there is just this one specific use case. 

I don't see us sorting this the other way around in the future. I'd either keep it like this or if you feel strong about this, use an enum with the types `DESC` and ASC` (because the name already says what will be sorted here).

Reply:
cool, I understand your rational and it makes sense to me now. thanks for explaining